packages/eslint-plugin-khan/demo/.eslintrc.js

```diff
@@ -1,6 +1,7 @@
-/* eslint-disable import/no-commonjs */
 module.exports = {
+    root: true,
```

Comment:
We set this so that the demo eslint config isn't merged with parent eslint config files!